conjur-oss/values.schema.json

```diff
@@ -0,0 +1,234 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "required": [
+    "database",
+    "dataKey",
+    "ssl"
+  ],
+  "properties": {
+    "account": {
+      "type": "string"
+    },
+    "affinity": {
+      "type": "object"
+    },
+    "authenticators": {
+      "type": "string"
+    },
+    "conjurLabels": {
+      "type": "object"
+    },
+    "database": {
+      "required": [
+        "ssl"
```

Comment:
The "ssl" field isn't required. If you don't provide this, the chart will create an SSL cert for you.